app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1137,6 +1142,13 @@ class BrowserTabViewModel(
                 lastTrackingInfo.destinationUrl = url
             }
         }
+
+        trackingParameters.lastCleanedUrl?.let {
+            trackingParameters.lastCleanedUrl = null
+            site?.urlParametersRemoved = true
```

Comment:
Once the page has loaded and a URL was cleaned, then it needs to set the `urlParametersRemoved` flag for the broken site reports. This is the best place to clear and set the broken site this since the page should have "settled" at this point.